modules/stanford_soe_helper_collection_layouts/stanford_soe_helper_collection_layouts.info

```diff
@@ -7,22 +7,34 @@ project = stanford_soe_helper_collection_layouts
 dependencies[] = bean
 dependencies[] = context
 dependencies[] = ctools
+dependencies[] = ds
 dependencies[] = features
 dependencies[] = image
+dependencies[] = options
 dependencies[] = pathauto
+dependencies[] = stanford_magazine
+dependencies[] = stanford_news_extras
+dependencies[] = stanford_people_spotlight
```

Comment:
@minorwm Do we need stanford_news_extras and stanford_people_spotlight as dependencies?